core/authHandler.js

```diff
@@ -21,6 +21,19 @@ var createHash = function (password) {
 	return bCrypt.hashSync(password, bCrypt.genSaltSync(10), null);
 }
 
+module.exports.isAdmin = function (req, res, next){
+	if(req.query.securityRating == 1) {
+		if(req.user.role=='admin')
+		{
+			next()
+		} else
+			res.status(401).send('Unauthorized')
+	}
+	else{
+		next()
+	}
+}
+
```

Comment:
Would it be better to take the security rating check out of this function? that way it can be more of a pure function that simply lets you know if the requester isAdmin, and the responsibility for checking the security rating is pushed to the calling function.

Reply:
thanks.  see new commits